microsoft/Multi-Agent-Custom-Automation-Engine-Solution-Accelerator
# docs: Add Comment Blocks for Multi-Agent API Endpoints

## Purpose

This PR adds detailed Swagger-style comment blocks for all key endpoints in the Multi-Agent API. These changes enhance the API's documentation by specifying parameters, request/response schemas, and descriptions for each endpoint, making the API more accessible and easier to use for developers and integrators.

## Does this introduce a breaking change?





## How to Test
* Get the code:

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code:
  1. Deploy the API to a local or test environment.
  2. Open the Swagger UI documentation interface (typically `/docs` or `/swagger`).
  3. Verify that the updated comment blocks render correctly with all intended parameters and response details.

## What to Check
Verify that the following are valid:
* Each endpoint displays its Swagger documentation properly in the UI.
* The added comment blocks align with the actual request and response schemas for the following endpoints:
  - `/input_task`
  - `/human_feedback`
  - `/human_clarification_on_plan`
  - `/approve_step_or_steps`
  - `/plans`
  - `/steps/{plan_id}`
  - `/agent_messages/{session_id}`
  - `/messages` (GET and DELETE)
  - `/api/agent-tools`

## Other Information

These changes are intended to improve developer experience and API usability. They do not modify the API's underlying functionality or introduce breaking changes.
